packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -57,7 +57,7 @@ void runPigeonIntegrationTests(TargetGenerator targetGenerator) {
       final HostIntegrationCoreApi api = HostIntegrationCoreApi();
 
       final AllTypes echoObject = await api.echoAllTypes(genericAllTypes);
-      compareAllTypes(echoObject, genericAllTypes);
+      expect(echoObject == genericAllTypes, true);
```

Comment:
Can't these all be `expect(foo, bar)` instead of `expect(foo == bar, true)`? The former should give better failure messages.

Reply:
As long as expect uses the built in equals method, yeah.

Reply:
Just realized this is in the integration tests changes, not the tests I wrote for this change. Will update.